### PR TITLE
Ibrahim onboarding

### PR DESCRIPTION
**Pull request by iakancay**

### CODE REVIEW

File: libs/ui/src/examples/Fetch/E6_RequestUseQuery.tsx [hunk 1]
```diff
@@ -36,11 +52,47 @@ export const RequestUseQuery = () => {
     <Box>
       <HStack>
         {/* TODO: LanguageSwitcher */}
+        {['en', 'nl', 'tr'].map(lang => (
```
Comment (7alip) [inline libs/ui/src/examples/Fetch/E6_RequestUseQuery.tsx:55]: You might haven already noticed that we have `LanguageSwitcher` and `LocaleSwitcher` components.

**Review by 7alip — APPROVED** [review]
It looks great, thanks for the submission.

**Comment by iakancay** [comment]
Thank you for your time. I have also some issue with styling i guess. :-)

**Comment by 7alip** [comment]
> Thank you for your time. I have also some issue with styling i guess. :-)

Styling is also important but not the main focus of this PR :)

**Pull request closed without merging**